src/ray/core_worker/core_worker.cc

```diff
@@ -436,6 +436,7 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
   };
   task_manager_.reset(new TaskManager(
       memory_store_, reference_counter_,
+      /*retry_task_callback*/
       [this](TaskSpecification &spec, bool delay) {
```

Comment:
```suggestion
      /* retry_task_callback= */[this](TaskSpecification &spec, bool delay) {
```

Reply:
clang-format just won't put them on the same line lol